app/presenters/concerns/assessment_tasks/review_documents_presenter.rb

```diff
@@ -0,0 +1,62 @@
+# frozen_string_literal: true
+
+module AssessmentTasks
+  extend ActiveSupport::Concern
+
+  class ReviewDocumentsPresenter
+    include Presentable
+
+    def initialize(template, planning_application)
+      @planning_application = planning_application
+      @template = template
+      @status = planning_application.review_documents_for_recommendation_status
+    end
+
+    def task_list_row
+      html = tag.span class: "app-task-list__task-name" do
+        concat review_documents_link
+      end
+
+      html.concat review_documents_status_tag
+    end
+
+    private
+
+    attr_reader :status
+
+    def review_documents_link
+      link_to(
+        "Review documents for recommendation",
+        planning_application_review_documents_path(planning_application), class: "govuk-link"
+      )
+    end
+
+    def review_documents_status_tag
+      classes = ["#{govuk_tag_class} app-task-list__task-tag"]
+
+      tag.strong class: classes do
+        I18n.t("status_tag_component.#{status}")
+      end
```

Comment:
I assume the above block creates a "strong" tag, inserts class of "classes" into it and nests the localised string inside? 🤔

Reply:
Yeah so the html formed would be something like
```
<strong class="govuk-tag govuk-tag--blue app-task-list__task-tag">Complete</strong>
```